src/main/java/health/ere/ps/service/connector/endpoints/EndpointDiscoveryService.java

```diff
@@ -34,13 +34,26 @@
      * Certificate to authenticate at the connector.
      */
     @ConfigProperty(name = "connector.cert.auth.store.file")
-    Optional<String> connectorTlsCertTrustStore;
+    Optional<String> connectorTlsCertAuthStoreFile;
 
     /**
      * Password of the certificate to authenticate at the connector.
      * The default value is a empty sting, so that the password must not be set.
      */
     @ConfigProperty(name = "connector.cert.auth.store.file.password", defaultValue = "!")
+    String connectorTlsCertAuthStorePwd;
+
+    /**
+     * Certificate to validate with the connector.
+     */
+    @ConfigProperty(name = "connector.cert.trust.store.file")
+    Optional<String> connectorTlsCertTrustStoreFile;
```

Comment:
Like i said before, i dislike ConfigProperties in different locations because of the possible double binding and interpetation and weak typing.